.github/workflows/docker-build-publish.yml

```diff
@@ -20,3 +20,11 @@ jobs:
     uses: celestiaorg/.github/.github/workflows/reusable_dockerfile_pipeline.yml@v0.2.2 # yamllint disable-line rule:line-length
     with:
       dockerfile: Dockerfile
+
+  docker-txsim-build:
+    permissions:
+      contents: write
+      packages: write
+    uses: celestiaorg/.github/.github/workflows/reusable_dockerfile_pipeline.yml@v0.2.2
+    with:
+      dockerfile: docker/Dockerfile_txsim
```

Comment:
Any reason we would just combine and put as a matrix?
```yml
  docker-security-build:
    strategy:
      matrix:
        dockerfile: [Dockerfile, docker/Dockerfile_txsim]
    permissions:
      contents: write
      packages: write
    uses: celestiaorg/.github/.github/workflows/reusable_dockerfile_pipeline.yml@v0.2.2
    with:
      dockerfile: ${{ matrix.dockerfile }}
```

REF: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#defining-the-maximum-number-of-concurrent-jobs

Github will try and run them in parallel, so unless there is a bug this is trying to work around, making them separate doesn't speed things up.

Reply:
also can be a follow up, doesn't need to block in PR. 👍